components/org.wso2.carbon.identity.api.server.api.resource/org.wso2.carbon.identity.api.server.api.resource.v1/src/main/resources/APIResources.yaml

```diff
@@ -481,8 +481,8 @@ components:
         type: integer
       example: 10
 
-    requiredAttributes:
-      name: requiredAttributes
+    attributes:
+      name: attributes
       in: query
       required: false
       description: Specifies the required parameters in the response. This parameter is not supported yet
```

Comment:
```suggestion
      description: Specifies the required attributes in the response. This parameter is not supported yet
```

Reply:
seems that with this effort we are supporting a few attributes. mention them in this description

Reply:
Fixed with https://github.com/wso2/identity-api-server/pull/508/commits/a554f52ff09da65532f69df35742292f62212fe4